_pages/en/strategy-build-vs-buy.md

```diff
@@ -160,24 +161,25 @@ The assumed reasoning behind this overarching strategy [TBC] is that by leveragi
 
 This oversimplified approach unfortunately doesn't take in account the fact that as an organization, ESDC is subject to multiple and fast paced changes, whether at the legal, political, organizational or technological levels.
 
-As such, it is critical for the organization to ensure that core to mandate IT Solutions remain flexible to adjust to the fast-paced changes while also minimizing the risk of said changes by adopting loosely coupled architectures and smaller sized work packages (see [Target Solution Delivery Model](strategy-target-solution-delivery-model.html)).
+As such, it is critical for the organization to ensure that core to mandate IT Solutions remain highly flexible and adaptable to adjust to the fast-paced changes while also minimizing the risk of said changes by adopting loosely coupled architectures and smaller sized work packages (see [Target Solution Delivery Model](strategy-target-solution-delivery-model.html)).
 
 ### Build vs Buy : Oversimplified Dichotomy<!-- omit in toc -->
 
 Looking for a systemic and overarching answer and approach to the question "Should the IT Solution be built or bought?" at the organizational level means bypassing the concept of solution architecture, where the required experts of the organization, both from business lines, finance, security, information technology, etc., come together to figure out the best possible way of accomplishing its mission through leveraging its various resources including people, tools, processes and rules.
```

Comment:
Perhaps I don't understand the corporate jargon here. What does "bypassing the concept of solution architecture" mean. Regardless of the solution, whether built or bought you need to think about how does it fit in to the other systems and services that it needs to talk too ? Or are we talking dismantling the practice to excessively plan out a whole system before any work is done ?

Reply:
We're saying that asking the question Build vs Buy as an overarching principle before doing any further analysis and user research equates bypassing the solutions architecture.

Also, we don't propose making the solutions architecture a central decision like it currently is the case at the moment. Let's not add more gates but instead make it easier for all product teams (in a DevOps model) able to take on their own architectural decisions.

Probably need to work on syntax... :thinking:

Reply:
"Looking for a one-size fits all answer to the Buy vs Build question" perhaps?